README.md

```
@@ -1 +1,100 @@
+## Testing
+
+To test the MCP server and verify it's working:
+
+1. **First, start the server** (in one terminal):
+   ```bash
+   uv run pr-inspector
+   ```
+   The server will start on `http://127.0.0.1:8000/mcp`
+
+2. **Then, run the test client** (in another terminal):
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)